addons/point_of_sale/models/pos_config.py

```diff
@@ -106,7 +106,7 @@ def _get_group_pos_user(self):
     session_ids = fields.One2many('pos.session', 'config_id', string='Sessions')
     current_session_id = fields.Many2one('pos.session', compute='_compute_current_session', string="Current Session")
     current_session_state = fields.Char(compute='_compute_current_session')
-    number_of_opened_session = fields.Integer(string="Number of Opened Session", compute='_compute_current_session')
+    number_of_rescue_session = fields.Integer(string="Number of Rescue Session", compute="_compute_current_session")
```

Comment:
```suggestion
    number_of_rescue_session = fields.Integer(string="Number of Rescue Session", compute='_compute_current_session')
```
Rule of thumb at Odoo, when developing in Python we're using double quotes `"` for string that should be displayed (message shown to the end-user, logs, etc) and single one `'` for technical string